helix-term/Cargo.toml

```diff
@@ -5,6 +5,7 @@ description = "A post-modern text editor."
 authors = ["Blaž Hrastnik <blaz@mxxn.io>"]
 edition = "2018"
 license = "MPL-2.0"
+include = ["src/**/*", "README.md"]
```

Comment:
I think this needs to include runtime/ somewhere so it can be installed with embed_runtime

Reply:
You're right!